helper.go

```diff
@@ -581,6 +581,9 @@ func parseLength(bytes []byte) (length int, cursor int) {
 		cursor += 2
 	default:
 		numOctets := int(bytes[1]) & 127
+		if len(bytes) < numOctets {
```

Comment:
Perhaps we also need to return an err from parseLength in the future, but silently failing is not preferred. Could you please revert this?

Reply:
I'm not sure about this one. This error crash our program on some use cases. But it was before the error check. 
We could do some logging, so it will not really be silent.

For now i will revert, because it can be resolved with the error checking. I will open a new PR if we have a crash